### PR TITLE
Properly initialize the rabbit @Rule

### PR DESCRIPTION
**Pull request by ericbottard**
Fixes spring-cloud/spring-cloud-stream-modules#164

### CODE REVIEW

**Comment by artembilan** [comment]
The `RabbitSourceInvalidConfigTests` should be fixed as well.

**Comment by ericbottard** [comment]
> The RabbitSourceInvalidConfigTests should be fixed as well.

Actually, none of those get a chance to hit rabbit

**Comment by ericbottard** [comment]
Updated my PR

File: rabbit-source/src/test/java/org/springframework/cloud/stream/module/rabbit/source/RabbitSourceInvalidConfigTests.java [hunk 1]
```diff
@@ -38,8 +38,8 @@
  */
 public class RabbitSourceInvalidConfigTests {
 
-	@Rule
-	public RabbitTestSupport rabbitAvailable;
+	@ClassRule
+	public static RabbitTestSupport rabbitAvailable = new RabbitTestSupport();
```
Comment (artembilan) [inline rabbit-source/src/test/java/org/springframework/cloud/stream/module/rabbit/source/RabbitSourceInvalidConfigTests.java:42]: Well, looking closely to this test, I'd say that we really don't need this rule here.
Our `Config` below is based only on the `RabbitSourceProperties`.

@ericbottard , you are right: it doesn't have hooks to try to access to Rabbit.

I can address that on merge.

**Comment by artembilan** [comment]
Merged as https://github.com/spring-cloud/spring-cloud-stream-modules/commit/9a52ea666f48b843e4f92eea76037eafc0b50c84

**Pull request closed without merging**